examples/helloworld/src/index.js

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import _ from 'underscore';
 import App from './App';
 
-React.render(<App />, document.getElementById('root'))
+
+var state = {
+    very: {
+        deeply: {
+            nested: {
+                counts: _.range(400).map(function () { return 0; })
+            }
+        }
+    }
+};
+
+function swapper(f) {
+    state = f(state);
```

Comment:
It is a bad idea to use the outer `state` object to manage component state, Because if the outer state object is modified by somewhere else, the Cursor to state mapping will be wrong, aka, the Cursor lost the control of the state object and cannot do immutable optimization.